#!/usr/bin/env python

head_joints = [
    'head_sidetilt_joint',
    'head_pan_joint',
    'head_tilt_joint',
    ]

right_arm_joints = [
    'right_arm_shoulder_lift_joint',
    'right_arm_elbow_rotate_joint',
    'right_arm_elbow_bend_joint',
    'right_arm_wrist_rotate_joint',
    'right_arm_gripper_finger_joint',
    'right_arm_shoulder_rotate_joint',
    ]

left_arm_joints = [
    'left_arm_shoulder_lift_joint',
    'left_arm_elbow_rotate_joint',
    'left_arm_elbow_bend_joint',
    'left_arm_wrist_rotate_joint',
    'left_arm_gripper_finger_joint',
    'left_arm_shoulder_rotate_joint',
    ]

all_joints = head_joints + right_arm_joints + left_arm_joints



